web3 module fails to build due to missing dependency

### Description

hedera-mirror-web3 fails to build due to the transient dependency `com.swirlds:swirlds-common:jar:0.31.0-alpha.1` not found

```
[INFO] -------------------< com.hedera:hedera-mirror-web3 >--------------------
[INFO] Building Hedera Mirror Node Web3 0.68.0-SNAPSHOT                   [3/3]
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.swirlds:swirlds-common:jar:0.31.0-alpha.1 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Hedera Mirror Node 0.68.0-SNAPSHOT:
[INFO]
[INFO] Hedera Mirror Node ................................. SUCCESS [  1.007 s]
[INFO] Hedera Mirror Node Common .......................... SUCCESS [  4.399 s]
[INFO] Hedera Mirror Node Web3 ............................ FAILURE [  0.384 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.346 s (Wall Clock)
[INFO] Finished at: 2022-10-21T15:46:48-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project hedera-mirror-web3: Could not resolve dependencies for project com.hedera:hedera-mirror-web3:jar:0.68.0-SNAPSHOT: com.swirlds:swirlds-common:jar:0.31.0-alpha.1 was not found in https://hyperledger.jfrog.io/artifactory/besu-maven/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of besu-repository has elapsed or updates are forced -> [Help 1]
```

### Steps to reproduce

as the description

### Additional context

_No response_

### Hedera network

other

### Version

v0.68.0-SNAPSHOT

### Operating system

_No response_

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -55,7 +55,6 @@ dependencies {
         api("com.playtika.testcontainers:embedded-postgresql:$testcontainersSpringBootVersion")
         api("com.playtika.testcontainers:embedded-redis:$testcontainersSpringBootVersion")
         api("com.salesforce.servicelibs:reactor-grpc-stub:$reactorGrpcVersion")
-        api("com.swirlds:swirlds-common:0.31.0") // workaround for hedera-mirror-web3 build failure
         api("com.vladmihalcea:hibernate-types-55:2.19.2")
         api("commons-beanutils:commons-beanutils:1.9.4")
         api("commons-io:commons-io:2.11.0")

--- a/buildSrc/src/main/kotlin/java-conventions.gradle.kts
+++ b/buildSrc/src/main/kotlin/java-conventions.gradle.kts
@@ -40,6 +40,12 @@ repositories {
     maven {
         url = uri("https://hyperledger.jfrog.io/artifactory/besu-maven/")
     }
+    maven {
+        url = uri("https://us-maven.pkg.dev/swirlds-registry/maven-prerelease-channel")
+    }
+    maven {
+        url = uri("https://us-maven.pkg.dev/swirlds-registry/maven-develop-snapshots")
+    }
 }
 
 dependencyManagement {

--- a/hedera-mirror-web3/pom.xml
+++ b/hedera-mirror-web3/pom.xml
@@ -106,12 +106,6 @@
             <artifactId>hedera-evm-api</artifactId>
             <version>${hedera-evm.version}</version>
         </dependency>
-        <!-- workaround for hedera-mirror-web3 build failure -->
-        <dependency>
-            <groupId>com.swirlds</groupId>
-            <artifactId>swirlds-common</artifactId>
-            <version>0.31.0</version>
-        </dependency>
 
         <!-- Test -->
         <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -152,6 +152,14 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>swirlds-prerelease</id>
+            <url>https://us-maven.pkg.dev/swirlds-registry/maven-prerelease-channel</url>
+        </repository>
+        <repository>
+            <id>swirlds-develop-snapshots</id>
+            <url>https://us-maven.pkg.dev/swirlds-registry/maven-develop-snapshots</url>
+        </repository>
     </repositories>
 
     <dependencies>

